ecms_base/features/custom/ecms_event/config/install/field.field.node.event.field_event_virtual_meeting_url.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_event_virtual_meeting_url
+    - node.type.event
+  module:
+    - link
+id: node.event.field_event_virtual_meeting_url
+field_name: field_event_virtual_meeting_url
+entity_type: node
+bundle: event
+label: 'Virtual Meeting URL'
+description: '(Optional) Add a virtual link URL if one exists.'
+required: false
+translatable: false
```

Comment:
Thinking this should be true in case they wanted to link to an alternate page/file, etc.